### ISSUE TITLE
ios device cannot run

### ISSUE THREAD

**Issue by GarkiOS** [SandeepAggarwal/CryptoppECC#7]
The emulator can run and install, everything is normal, but when I switch to the device, it starts to report errors, is there any good way?

**Comment by merry37** [comment]
yes same issue i faced  its running fine on simulator but when i switched it to device Liker error occurs
**building for iOS, but linking in object file built for iOS Simulator, for architecture arm64**

**Comment by Mtczf** [comment]
> yes same issue i faced its running fine on simulator but when i switched it to device Liker error occurs **building for iOS, but linking in object file built for iOS Simulator, for architecture arm64**

I encountered the same problem as you, how did you solve it in the end?
